CONFIGURE.md

```diff
@@ -166,6 +166,25 @@ folder. You may specify your own path in place of these defaults.
      nifi.flowfile.repository.directory.default=${MINIFI_HOME}/flowfile_repository
 	 nifi.database.content.repository.directory.default=${MINIFI_HOME}/content_repository
 
+It is also possible to use a single database to store multiple repositories with the `minifidb://` scheme.
+This could help with migration and centralize agent state persistence.
+
+     in minifi.properties
+     nifi.flowfile.repository.directory.default=minifidb://${MINIFI_HOME}/agent_state/flowfile
+	 nifi.database.content.repository.directory.default=minifidb://${MINIFI_HOME}/agent_state/content
+	 nifi.state.manangement.provider.local.path=minifidb://${MINIFI_HOME}/agent_state/processor_states
```

Comment:
What does the `minifidb://` scheme exactly mean? Does it interpret the last path segment as a column family and the segments before as a path to the database?